plugins/vpp/model/interfaces/interfaces.proto

```diff
@@ -67,9 +67,11 @@ message Interfaces {
         Memif memif = 101;
 
         message Vxlan {
-            string src_address = 1;  /* source VTEP address */
-            string dst_address = 2;  /* destination VTEP address */
-            uint32 vni = 3;          /* VXLAN Network Identifier */
+            string multicast = 1;    /* multicast interface */
```

Comment:
Order should not be changed if possible. This makes this change **backwards incompatible**, maybe not for JSON, but for other possible formats it does.

Reply:
good point